db/schema.rb

```diff
@@ -1,4 +1,3 @@
-# typed: strict
```

Comment:
Can you restore this? Or would you prefer to handle otherwise?

Reply:
Sure, this was just db:migrate + rubocop -a that made things this way.